openpgp/write.go

```diff
@@ -469,7 +469,7 @@ func AttachedSign(out io.WriteCloser, signed Entity, hints *FileHints,
 		}
 	}
 
-	hasher := crypto.SHA512
+	hasher := config.Hash() // defaults to SHA-256
```

Comment:
should we upgrade config.Hash to default to SHA512? KBPGP uses SHA512 almost everywhere. cc @maxtaco